lib/src/room.dart

```diff
@@ -972,6 +976,12 @@ class Room {
         ? await client.encryption!
             .encryptGroupMessagePayload(id, content, type: type)
         : content;
+
+    final utf8EncodedJsonString = utf8.encode(sendMessageContent.toString());
```

Comment:
Shouldn't this use jsonEncode() and then utf8 encode it? This will calculate the size of whatever the dart representation is, when printing it, doesn't it?

Reply:
correct thanks fixed